handbook/product/design/onboarding/index.md

```diff
@@ -1,3 +1,26 @@
 # Onboarding
 
-(Coming soon!)
+Welcome to the Design Team!
+
+Havng gone though [the general onboarding](https://about.sourcegraph.com/handbook/people-ops/onboarding/general_onboarding) and [the Product onboarding](https://about.sourcegraph.com/handbook/product/onboarding), it's time for the Design onboarding!
+
+Your perspective as both a new user of Sourcegraph and a new teammate is very valuable to us. Please keep notes on any issues you encounter as you are onboarding and learning the product. We’ll use those notes to improve the product and process.
+
+## Designer checklist
+
+- Add your name to the [Project card](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/🛠Project-Tools?node-id=1%3A203) designer list
+- Create your first Figma project
```

Comment:
The new designer may not be actively working on a feature or project during the first week or two -- Did we just want to leave this as "Get familiar with Project Tools, Templates, and Wildcard"?

Reply:
I've moved the 'first figma project' to the separate section so it's not a part of the checklist but an independent item